include/behaviortree_cpp/utils/safe_any.hpp

```diff
@@ -269,15 +276,15 @@ inline Any &Any::operator =(const Any &other)
 
 inline bool Any::isNumber() const
 {
-  return _any.type() == typeid(int64_t) ||
-         _any.type() == typeid(uint64_t) ||
-         _any.type() == typeid(double);
+  return type() == typeid(int64_t) ||
+         type() == typeid(uint64_t) ||
+         type() == typeid(double);
```

Comment:
Not sure about this. Shouldn't you include all the possible arithmetic types?

Reply:
Or use `_any.type()`?

Now the `isNumber` function is not called in behaviortree_cpp (`isCastedAsNumber` is used instead).